src/services/ui/src/components/Layout/index.tsx

```diff
@@ -49,7 +44,7 @@ const UserDropdownMenu = () => {
   const navigate = useNavigate();
 
   const handleViewProfile = () => {
-    navigate("/profile");
+    navigate({ path: "/profile" });
```

Comment:
`useNavigate` is now TS wrapped.  It accepts a new parameter type - obj that enforces type safety.

<img width="1146" alt="Screenshot 2023-12-22 at 11 21 36 AM" src="https://github.com/Enterprise-CMCS/macpro-mako/assets/141361476/fedd8930-81ce-4008-b857-e424aa851beb">

In addition, it infers paths with wildcard id parameters:

<img width="1071" alt="Screenshot 2023-12-22 at 11 22 04 AM" src="https://github.com/Enterprise-CMCS/macpro-mako/assets/141361476/9309faca-f702-4340-aeb5-4b10fc6a83b8">